components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,291 +10,788 @@ toc = true
 
 ### Updating a Policy
 
-In the IAM beta, policies must be updated from the command line.
+```bash
+curl -sSH "api-token: $TOKEN" -d @policy.json -X PUT \
+https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}?pretty
+```
 
-* You can modify the *membership* of both Chef-managed and custom policies; see [Policy Membership]({{< relref "iam-v2-api-reference.md#policy-membership" >}}) for more information.
-* You can modify the *definition* of custom policies but not Chef-managed policies.
-* You can change the policy name, members, or statements, but you cannot change the policy ID.
-* Modifying any part of a policy resets *all* of its properties.
-  Properties that aren't included in the command are reset to empty values.
-  Thus, you must supply all of a policy's properties, not just the ones you wish to update.
+- Supply all of a policy's properties, not just the ones you wish to update.
+  Properties that you do not include are reset to empty values.
+- You can modify the *membership* of both Chef-managed and custom policies.
+- You can modify the *definition* of custom policies but not Chef-managed policies.
+- The policy ID is immutable; it can only be set at creation time.
 
-In order to update a policy, you must know its ID.
-Use the command from [Listing Policies]({{< relref "iam-v2-api-reference.md#listing-policies" >}}) to locate your policy ID.
+### Deleting a Policy
 
-The simplest way to pass a policy is by storing it in a file; we used a sample `policy.json` in the previous section.
-To update that policy you still need to pass in the complete policy definition--plus its members--just as you did when creating it; any omitted properties will be erased from the policy!
+```bash
+curl -sSH "api-token: $TOKEN" -X DELETE \
+https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}?pretty
+```
+
+Deleting a policy is permanent and cannot be undone.
+
+## Policy Membership
+
+Chef Automate supports two methods for updating policy membership.
+
+Using `PUT` lets you set membership for the entire policy; see [Updating All Members On a Policy]({{< relref "iam-v2-api-reference/#updating-all-members-on-a-policy" >}}).
+
+Using `POST` lets you add members to (or remove members from) an existing membership list.
+For more information, see [Adding Members to a Policy]({{< relref "iam-v2-api-reference/#adding-members-to-a-policy" >}}) and [Removing Members from a Policy]({{< relref "iam-v2-api-reference/#removing-members-from-a-policy" >}}).
 
-As an example, to update the example policy to keep the first policy statement allowing users list-access, while removing `editor` access, edit your `policy.json` file:
+HTTP request                           | Description
+---------------------------------------|------------
+GET /policies/{**id**}/members         | lists the membership
+PUT /policies/{**id**}/members         | replaces the membership
+POST /policies/{**id**}/members:add    | add to the membership
```

Comment:
nit: `lists` and `replaces` are plural, but `add` and `remove` are not. Could make them all singular.